packages/lib/server/repository/membership.ts

```diff
@@ -354,6 +354,21 @@ export class MembershipRepository {
       },
     });
   }
+  static async findAllAcceptedPublishedTeamMemberships(userId: number, tx?: PrismaTransaction) {
```

Comment:
let's remove `findAllAcceptedMemberships` and use this function everywhere instead